src/OpenIddict.Abstractions/Primitives/OpenIddictConverter.cs

```diff
@@ -23,10 +23,7 @@ public class OpenIddictConverter : JsonConverter
         /// <returns><c>true</c> if the type is supported, <c>false</c> otherwise.</returns>
         public override bool CanConvert([NotNull] Type type)
         {
-            if (type == null)
-            {
-                throw new ArgumentNullException(nameof(type));
-            }
+            type = type ?? throw new ArgumentNullException(nameof(type));
```

Comment:
I'm not a huge fan of this syntax, which also produces a different IL output (https://sharplab.io/#v2:C4LglgNgNAJiDUAfAAgJgIwFgBQyDMABGgQMI4DeOB1RhyALAQLIAUAysAE5gB2A5gQAOAQ07CAtgFNgkzgEoqNSthqqCYAGYEWIsVJmcCAXiMEeAVwgQFKtdWV27wABacA9gHczkrwEFOfOZSPMAAcpYQAKIAHgDGkoLAYG48LDwSkm4aOqIZBnJyANyKdgC+JQTlttQVFfhEjEyo7Fy8Arp5sjaqDo4d+rLGQrkDhgD8YwQu7l48PgT+gcFhETHxicmp6VJZOXrSXUUVVaVAA=)

Any chance you could revert these changes? This should help make your PR much smaller.

Reply:
Rollbacking changes